### PR TITLE
Fixed method documentation issues

### LINKED ISSUE #504: Documentation issues.
There are a few issues with the current documentation which are listed below.
- [collection/certificate.js line 20](http://www.postmanlabs.com/postman-collection/collection_certificate.js.html#line20) - `Typo` {{ src: (String) }} [cert] Object with path on the file system **fot** certificate file, as src. 

- [Certificate List example](http://www.postmanlabs.com/postman-collection/CertificateList.html) - Doesn't `require('postman-collection').CertificateList` before using in the example.

- [Load collection example](http://www.postmanlabs.com/postman-collection/Collection.html) Calls `inspect` function, which is undefined, instead of `pretty`.

- [Write collection to file example](http://www.postmanlabs.com/postman-collection/Collection.html) The `toString()` method writes the collection as `[object Object]` rather than actual data.

- [Collection variable example](http://www.postmanlabs.com/postman-collection/Collection.html) The `{{apiBaseUrl}}` doesn't resolve to the variable value in the request.

- [collection/cookie.js line 32](http://www.postmanlabs.com/postman-collection/collection_cookie.js.html#line32) `Typo` {String=} [expires] Expires sets an expiry date for when a **coo0kie** gets deleted. It should either be a

- [Cookie create example](http://www.postmanlabs.com/postman-collection/Cookie.html) Missing comma after maxAge property while instantiating `myCookie`, invalid JSON definition of **cookie**(instead of collection) at bottom of page. 

-  [Header parse example](http://www.postmanlabs.com/postman-collection/Header.html) Assert not defined, strings not being trimmed before matching.

- [Create header example](http://www.postmanlabs.com/postman-collection/Header.html) `Create a Header` added in the code example block.

**Edit 1** 

- [Add item to collection example](http://www.postmanlabs.com/postman-collection/Item.html) - Missing `items` in adding to collection syntax(should be `myCollection.items.one("my-folder-1").items.add`), quotes mismatch on `request.url`.

- [Add script to collection example](http://www.postmanlabs.com/postman-collection/Item.html) - Missing `,` after `"id": "my-get-request"`, quotes mismatch in `request.url`. 

**Edit 2** 

- [Add itemGroup to collection example](http://www.postmanlabs.com/postman-collection/ItemGroup.html) - Missing `items` in adding to collection syntax(should be `myCollection.items.add(new ItemGroup({..})`).

- [Sample itemList](http://www.postmanlabs.com/postman-collection/ItemGroup.html) - Missing quote end in `script` key.

- [ProxyConfig JSON](http://www.postmanlabs.com/postman-collection/ProxyConfig.html) - Invalid JSON, property keys not quoted, missing `,` after `port` key.

- [VariableScope iteration Example](http://www.postmanlabs.com/postman-collection/VariableScope.html) - Doesn't `require('postman-collection').VariableScope` before using in the example.

Will keep updating this as new issues are found.

### PR DESCRIPTION
**Pull request by deepakpathania**
Fixes #504.

### CODE REVIEW

**Comment by shamasis** [comment]
@kunagpal - the build of appveyor failed on the RegExp security check timeout in Node 8. Odd.

**Comment by kunagpal** [comment]
@shamasis I've restarted it for now.

File: lib/collection/certificate-list.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ _.inherit((
      * @param {Array} list - The list of certificate representations
      *
      * @example <caption>Create a new CertificateList</caption>
-     * var certificateList = new CertificateList({}, [
+     * var CertificateList = require('postman-collection').certificateList,
```
Comment (kamalaknn) [inline lib/collection/certificate-list.js]: should be `require('postman-collection').CertificateList`

Comment (deepakpathania) [reply lib/collection/certificate-list.js]: Fixed in #507.

**Pull request merged**